gt/main.c

```diff
@@ -633,6 +732,16 @@ cleanup_gt_instance(struct gt_instance *instance)
 {
 	lua_close(instance->lua_state);
 	instance->lua_state = NULL;
+
+	if (instance->neigh.cache_tbl) {
```

Comment:
Need to add an explicit check for `!= NULL` by the standards of DPDK. Same for the hash table test below.